### PR TITLE
images: Update gcb-docker-cloud image to v20210722-085d930

### PR DESCRIPTION
**Pull request by justaugustus**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

images: Update gcb-docker-cloud image to v20210722-085d930

Signed-off-by: Stephen Augustus <foo@auggie.dev>

#### Does this PR introduce a user-facing change?

```release-note
images: Update gcb-docker-cloud image to v20210722-085d930
```

### CODE REVIEW

**Review by cpanato — APPROVED** [review]
lgtm

**Comment by cpanato** [comment]
adding hold to make sure all jobs pass, will lift the hold after that

/hold

**Review by saschagrunert — APPROVED** [review]

File: dependencies.yaml [hunk 1]
```diff
@@ -424,3 +424,22 @@ dependencies:
     refPaths:
     - path: images/build/setcap/variants.yaml
       match: '[a-zA-Z]+\-v((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)'
+
+# Build environments
+  - name: "gcr.io/k8s-testimages/gcb-docker-gcloud"
+    version: v20210722-085d930
+    refPaths:
+    - path: cmd/vulndash/cloudbuild.yaml
+      match: "name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v([0-9]{8})-([a-z0-9]{7})'"
```
Comment (saschagrunert) [inline dependencies.yaml:433]: nit, the groups are not required:
```suggestion
    - path: cmd/vulndash/cloudbuild.yaml
      match: "name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v[0-9]{8}-[a-z0-9]{7}'"
```

**Review by saschagrunert — APPROVED** [review]

**Comment by justaugustus** [comment]
(don't `/retest`; the jobs are likely exhausted)

File: dependencies.yaml (hunk 1 above)
Comment (justaugustus) [reply dependencies.yaml]: Great point! Fixed!

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**